.github/workflows/test.yml

```diff
@@ -28,6 +29,10 @@ jobs:
         with:
           key: "${{ matrix.os }}-rust-cache"
 
+      - name: Install wash
+        run: |
+          cargo install --version ${{ matrix.wash_version }} wash-cli
```

Comment:
@brooksmtownsend Can we apt install here or will it give us a version that won't work on ubuntu 22?

@vados-cosmonic for reference, this is what we have available: https://github.com/wasmCloud/common-actions/blob/main/install-wash/action.yml. If we can use it, this pipeline will run much more quickly. Otherwise, what you have will work

Reply:
We can do apt here!